### PR TITLE
Add `errorRecovery` option

### PR DESCRIPTION
**Pull request by ojthek**
The command line argument -fix is now not longer fixed. I made it optional, default is true, so the behavior is the same. 

Broken images will not longer return a "fixed" png, if I disable the option.

### CODE REVIEW

**Comment by ojthek** [comment]
@sindresorhus Could you have look?

**Comment by sindresorhus** [comment]
The readme description needs to be clearer. Not clear what “image recovery” means.

**Comment by ojthek** [comment]
Hi @sindresorhus 
I added more comment, and changed the name of the option to be clearer.
Best regards

**Comment by ojthek** [comment]
Hi @sindresorhus, 
could you have look? Or could you say when you have time for it?

Best regards,
Oliver

**Comment by ojthek** [comment]
Hi @sindresorhus, 
could you have look? Or could you say when you have time for it?

Best regards,
Oliver

**Comment by sindresorhus** [comment]
> Broken images will not longer return a "fixed" png, if I disable the option.

Are you saying that your image was not fully fixed with this? If so, I think we should make it `false` by default as it could cause you to have broken images (almost fixed ones) without any feedback.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "imagemin-optipng",
-	"version": "7.0.0",
+	"version": "7.0.1",
```
Comment (sindresorhus) [inline package.json]: Don't bump this.

File: test.js [hunk 2]
```diff
@@ -27,3 +28,17 @@ test('colorTypeReduction option', async t => {
...
+	const data = await optipng()(fixtureBroken);
+	t.true(isPng(data));
+});
+
+test('recoverImage explicit', async t => {
+	const data = await optipng({errorRecovery: true})(fixtureBroken);
+	t.true(isPng(data));
+});
+
+test('recoverImage is set to false', async t => {
```
Comment (sindresorhus) [inline test.js]: Wrong option name

**Comment by ojthek** [comment]
> > Broken images will not longer return a "fixed" png, if I disable the option.
> 
> Are you saying that your image was not fully fixed with this? If so, I think we should make it `false` by default as it could cause you to have broken images (almost fixed ones) without any feedback.

You are right. I think so too, but for compatibility-reason I thought, it would be better if the library won't change it's behavior. So an update won't change the behavior of the projects, which already use this library. 

Otherwise the version-number should show the difference in the behavior.  If you want, I will change the default behavior. Please give me a short note.

Best regards

**Comment by ojthek** [comment]
Hi @sindresorhus,

could you have a look? :-)

Thank you in advance, best regards

**Comment by ojthek** [comment]
Hello @sindresorhus,
I hope you're feeling well. Could you have a look at this request? If not, could you tell me a date by which you will have the time to do? 

Best regards

**Pull request merged**